tidy.Rmd

```diff
@@ -120,89 +125,93 @@ The second step is to resolve one of two common problems:
 
-### Gathering
+### Longer
 
-A common problem is a dataset where some of the column names are not names of variables, but _values_ of a variable. Take `table4a`: the column names `1999` and `2000` represent values of the `year` variable, and each row represents two observations, not one.
+A common problem is a dataset where some of the column names are not names of variables, but _values_ of a variable. Take `table4a`: the column names `1999` and `2000` represent values of the `year` variable, the values in the `1999` and `2000` columns represent values of the `cases` variable, and each row represents two observations, not one.
 
 ```{r}
 table4a
 ```
 
-To tidy a dataset like this, we need to __gather__ those columns into a new pair of variables. To describe that operation we need three parameters:
+To tidy a dataset like this, we need to __pivot__ the offending columns into a new pair of variables. To describe that operation we need three parameters:
 
-* The set of columns that represent values, not variables. In this example, 
+* The set of columns whose names are values, not variables. In this example, 
   those are the columns `1999` and `2000`.
 
-* The name of the variable whose values form the column names. I call that
-  the `key`, and here it is `year`.
+* The name of the variable to move the column names to. Here it is `year`.
 
-* The name of the variable whose values are spread over the cells. I call 
-  that `value`, and here it's the number of `cases`.
+* The name of the variable to move the column values to. Here it's `cases`.
   
-Together those parameters generate the call to `gather()`:
+Together those parameters generate the call to `pivot_longer()`:
 
 ```{r}
 table4a %>% 
-  gather(`1999`, `2000`, key = "year", value = "cases")
+  pivot_longer(c(`1999`, `2000`), names_to = "year", values_to = "cases")
 ```
 
-The columns to gather are specified with `dplyr::select()` style notation. Here there are only two columns, so we list them individually. Note that "1999" and "2000" are non-syntactic names (because they don't start with a letter) so we have to surround them in backticks. To refresh your memory of the other ways to select columns, see [select](#select).
+`year` and `cases` do not exist in `table4a` so we put their names in quotes.
 
-```{r tidy-gather, echo = FALSE, out.width = "100%", fig.cap = "Gathering `table4` into a tidy form."}
+The columns to pivot are specified with `dplyr::select()` style notation. Here there are only two columns, so we list them individually. Note that "1999" and "2000" are non-syntactic names (because they don't start with a letter) so we have to surround them in backticks. To refresh your memory of the other ways to select columns, see [select](#select).
+
+`year` and `cases` do not exist in `table4a` so we put their names in quotes.
```

Comment:
Repetition of sentence above?

Reply:
Thanks for catching this! Removed in 27dfedf454afbae25a6d24f00c819bb1a9264a41

Reply:
BTW you can just tell me you did it; you don't need to prove it since I trust you 😄